src/include/network/lan/cmdline.rb

```diff
@@ -206,7 +206,7 @@ def update_builder_from_options!(builder, options)
       case builder.type.short_name
       when "bond"
         # change only if user specify it
-        builder.slaves = options["slaves"].split(" ") if options["slaves"]
+        builder.childs = options["slaves"].split(" ") if options["slaves"]
```

Comment:
I would prefer **members** than **childs** as commented in the feature, but it is just my POV. @mvidner what do you think?

Reply:
well ... I've intentionally used grammatically incorrect word "childs". In hope that it won't have some unexpected side meanings. 

At least it will be easy to change by one regexp to something else if not accepted ;-)

Reply:
See also http://lkml.iu.edu/hypermail/linux/kernel/2010.0/02186.html, looks like there is some request for using `port` as it is already used for bridges.

Reply:
> well ... I've intentionally used grammatically incorrect word "childs". In hope that it won't have some unexpected side meanings.
> 
> At least it will be easy to change by one regexp to something else if not accepted ;-)

I know but think that we have to also update the documentation so would like to at least share it with wicked team and others.

Reply:
> well ... I've intentionally used grammatically incorrect word "childs". In hope that it won't have some unexpected side meanings.
> 
> At least it will be easy to change by one regexp to something else if not accepted ;-)

That regexp would rename also the **VLAN** method that were already using **parent / child** terminology so that could be an unexpected side effect change :P but I'm being too nitpicking now :)

Reply:
> See also http://lkml.iu.edu/hypermail/linux/kernel/2010.0/02186.html, looks like there is some request for using `port` as it is already used for bridges.

IMHO it's good to follow the kernel side discussion of the renaming. Here's [version 4 (2020-11) of the above patch set][1] (which was v2, 2020-10), the best part is [5/5 which is the documentation][5].

[5]: https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2370989.html
[1]: https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2370988.html

Reply:
The kernel says:

> Major terminology rework done late 2020, to start to remove the use of
> the socially problematic terms "master" and "slave" from the code. The
> "master" device is now referred to as simply the "bond" device and the
> "slave" devices as "ports", but all sysfs, procfs and module options
> have been retained as-is for userspace compatibility...